node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala

```diff
@@ -373,5 +373,10 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
   }
   addSubcommand(propose)
 
+  val dump = new Subcommand("dump") {
+    descr("Dumps the current state of the Node to the file")
```

Comment:
Technically it just dumps all the blocks in the current main chain of the fork choice, so I would just say "Dumps the current state of the blockDAG to the file" (or chain).

Reply:
We are planned to dump more info like heapdump, metrics, the configuration used, so I think the "state" word reflects this correctly.

Reply:
Sounds good